syntax = "proto3";

package server_health;

// Message representing the server's health status
message ServerHealth {
  string host = 1;
  uint32 port = 2;
  uint32 sessions = 3;
  uint32 max_sessions = 4;
}